### DISCUSSION TITLE
Help with Repositories / Entities and Linq / query language???

### DISCUSSION THREAD

**Discussion by MomboMan** [fullstackhero/blazor-starter-kit#317] [Q&A]
Hi,

First off thanks for adding a discussion tab, I think this will help a lot.

Secondly, I need help, can someone explain how to combine what was built here and adding Linq or query language queries?  I'm wanting to run a join on two or three different tables.

In the Application project, using the current _unitOfWork stuff and then add something like this:
````    
        var data = await _unitOfWork.Repository<Employee>().Entities
        .Join(Person,
                Employee => e.Id,
                Person => p.Id,
                (e, p) => new {
                        Id = e.Id,
                        FirstName = p.firstName,
                        LastName = p.lastName,
                        City = p.City
                }
        )
        .Join(EmployeeSchedule,
                Employee => e.Id,
                EmployeeSchedule => es.EmployeeID,
                (e, es) => new {
                        DateAvailable = es.DateAvailable,
                        Available = es.Available
                }
        )
…
````

I have it working on the client side but I'd really like it hidden under the Application project.

Help? This isn’t my universe.

Thanks

Craig

**Comment by Zigimun** [comment]
Can you show the full file of the handler?

Best regards
